src/frontends/pytorch/src/node_context.cpp

```diff
@@ -180,6 +180,55 @@ std::string NodeContext::const_input<std::string>(size_t index) const {
     return input->get_decoder()->as_string();
 }
 
+namespace {
+template <typename T>
+Any get_constant_data(const std::shared_ptr<opset10::Constant>& constant) {
+    const T* ptr = reinterpret_cast<const T*>(constant->get_data_ptr());
+    const auto& shape = constant->get_shape();
+    if (is_scalar(shape)) {
+        return ptr[0];
+    }
+    return std::vector<T>(ptr, ptr + shape_size(shape));
+}
+}  // namespace
+
+Any NodeContext::const_input_as_any(int index) const {
```

Comment:
@mvafin could you take a look?
I'm not sure that we actually need this method.
There is a data field in python representation of ov::Constant which can be used instead of this
https://github.com/openvinotoolkit/openvino/blob/5e835e327b235b17e605de7f18eaf49a7c806bc7/src/bindings/python/src/pyopenvino/graph/ops/constant.cpp#L193

Reply:
I think we need this method to be able to get values of inputs, especially if those inputs are strings or nones. I would recommend rename it to the generic name that can be used in other frontends too, for example `get_input_const_value`, @itikhono do you agree?

Reply:
it looks like we can't re-use Constant class functionality.
`get_values_from_const_input` maybe?

Reply:
done